src/css/custom.css

```diff
@@ -171,49 +177,72 @@ svg.implemented:hover [stroke="#333"] {
   text-align: center;
 }
 
+/* Badge padding and styles */
 .badge {
-  margin-left: 5px;
+  margin-left: 0;
```

Comment:
blocking, I think: 

One place that is negatively affected by this change is the release notes page. Here's an example of a badge that now sits flush against the heading (on the left is without these changes, on the right is with): 

<img width="1755" alt="image" src="https://github.com/user-attachments/assets/0223bbf2-c8dc-4e83-93cc-3bad55f773d3">

Can we target these badges via a combination of something unique to these pages and `badge:first-of-type` to apply `left-margin` to only the first one? 

(I also notice that there are empty spaces at the end of a lot of headings which is also nudging the badges over, and that doesn't feel right to me...so applying the change I'm looking for might also include removing those ending spaces.)

Reply:
Fixed (hopefully) with the badge style changes - no more empty space to the right, and targeting the main badge style etc as suggested - agree much better - thank you!